### ISSUE TITLE
Regression: Black Steel Recipe Does not Work Again

### ISSUE THREAD

**Issue by lilpaladin1** [Nomi-CEu/nomi-ceu#310]
**Pack Version:**
1.6

**Launcher:**
Prism Launcher

**Mode:**
Normal

**Environment:**
Singleplayer

**Description:**
The Black Steel Dust/Fluid Recipe does not work again, it was previously fixed in #84 but with the newest pack release, it has broken again. It is again the NBT version of the Red Coal that is preventing the recipe from running, as using an NBT-less version of the Red Coal from the creative inventory works fine. This probably happened with the introduction of that Dark Red Coal version of the recipe in the Alloy Blast Smelter.

**Expected Behavior:**
The Black Steel Recipe on both the Mixer and Alloy Blast Furnace work.

**Steps To Reproduce:**
Put items in either Mixer or Alloy Blast Furnace.

**Crash Log:**
No Crashing

**Screenshots:**

![javaw_uSJ9ytQPTk](https://user-images.githubusercontent.com/17087669/230726752-8d12eeae-49cd-41c6-a77f-9a976b1cc648.png)

**Comment by IntegerLimit** [comment]
This is most likely a GT issue. Not sure though.

**Comment by IntegerLimit** [comment]
Yeah, I saw those errors too. Looking at it, and, Black Steel will be implemented in groovy, due to the NBT requirements.

**Comment by lstanden** [comment]
FYI, the following change fixed my mixer recipe:
```
Before:
<extrautils2:ingredients:4>

After:
<extrautils2:ingredients:4>.withEmptyTag()
```

**Comment by IntegerLimit** [comment]
Oh, really...

**Comment by bhodii** [comment]
I can confirm the above, in https://github.com/Nomi-CEu/nomi-ceu/blob/a2efbed8b3c289a57b7323a2bfcddee9ddcbb9f2/overrides/scripts/Earlygame.zs#L195 changing

`mixer.recipeBuilder().inputs([<ore:dustSteel> * 3,<ore:dustBlackBronze> * 2,<actuallyadditions:item_crystal:3> * 2,<extrautils2:ingredients:4> * 2]).outputs(<metaitem:dustBlackSteel>  * 9).EUt(15).duration(200).buildAndRegister();`

to

`mixer.recipeBuilder().inputs([<ore:dustSteel> * 3,<ore:dustBlackBronze> * 2,<actuallyadditions:item_crystal:3> * 2,<extrautils2:ingredients:4>.withEmptyTag() * 2]).outputs(<metaitem:dustBlackSteel>  * 9).EUt(15).duration(200).buildAndRegister();`

allows the mixer to mix black steel.

**Comment by IntegerLimit** [comment]
Fixed with https://github.com/Nomi-CEu/nomi-ceu/commit/7a13c1d24b8f5aa1a5d4e3bcc35aeb3a1631fb40.

**Comment by omletformage** [comment]
when will this be implemented?

**Comment by IntegerLimit** [comment]
Has been implemented. Black steel is fixed in 1.6.1 alpha.
